packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/java/enrollmentqrcodeintegrationsteps.md

```diff
@@ -0,0 +1,51 @@
+#### 1. Initiate sign-in flow and return a list of authenticators
```

Comment:
Sequence diagram missing before this step

Reply:
Fixed, the markdown build doesn't tell you when there are errors 😰

Reply:
```suggestion
#### 1. Start enrollment, retrieve and display authenticators
```